presto-tests/src/main/java/io/prestosql/tests/AbstractTestIntegrationSmokeTest.java

```diff
@@ -173,6 +173,13 @@ public void testSelectInformationSchemaColumns()
                 "SELECT table_name, column_name FROM information_schema.columns " +
                         "WHERE table_catalog = '" + catalog + "' AND table_schema = '" + schema + "' AND table_name LIKE '%orders%'",
                 ordersTableWithColumns);
+
+        assertQuerySucceeds("SELECT * FROM information_schema.columns");
```

Comment:
How does it test table disappearing?

Reply:
It's not disappearing. Look how `io.prestosql.metadata.MetadataManager#listTableColumns` works -- 
every catalog is actually 3 connectors. Each of them is tried in order to lists columns. Maybe, a different fix could be in that function, so that 
```
for (ConnectorId connectorId : catalogMetadata.listConnectorIds()) {
```
loops stops as soon as 
```
metadata.listTableColumns(connectorSession, tablePrefix)
```
returns something?

Reply:
> every catalog is actually 3 connectors
what are those connectors?
> loops stops as soon as
Maybe they all can return something useful about different tables?

Reply:
> what are those connectors?

information_schema connector, system tables connector and _the_ connector

> Maybe they all can return something useful about different tables?

 yes, especially when `tablePrefix` has schema set but not the table name